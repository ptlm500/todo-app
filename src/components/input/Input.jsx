import styled from 'styled-components';
import { theme } from '../../styles';

export default styled.input`
  background: ${theme.background};
  color: ${theme.text};
  font: inherit;
  appearance: none;
  border: none;
  border-bottom: 1px solid ${theme.divider}
`;
