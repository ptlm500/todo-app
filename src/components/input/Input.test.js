import React from 'react';
import renderer from 'react-test-renderer';
import Input from './Input';
import 'jest-styled-components';

describe('Input', () => {
  it('renders an input', () => {
    const tree = renderer.create(<Input />);
    expect(tree).toMatchSnapshot();
  });
});
