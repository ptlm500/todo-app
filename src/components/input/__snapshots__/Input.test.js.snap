// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders an input 1`] = `
.c0 {
  background: #FFFFFF;
  color: #212121;
  font: inherit;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  border-bottom: 1px solid #BDBDBD;
}

<input
  className="c0"
/>
`;
